backend/__tests__/controllers/houses.spec.ts

```diff
@@ -107,9 +107,9 @@ describe('/house routes', () => {
       .set(headers)
       .expect(200)
       .then(({ body }) => {
-        expect(body.length).toBe(1);
-        expect(body[0].default_ast).toBe(1);
-        expect(body[0].name).toBe('house name 1');
+        expect(typeof body).toBe('object');
```

Comment:
This should be avoided, as these "types" in JS aren't reliable. 
`typeof []` for example results in 'object' as well.

Reply:
Or are you going by the "I tested for both the type and the length so it's an array" approach? 😁

Reply:
I'll just delete that line then.